app/routes/index.js

```diff
@@ -0,0 +1,25 @@
+// internal modules
+const fs = require('fs');
+
+// 3rd party modules
+const logger = require('winston');
+
+function registerRoutes(app)
+{
```

Comment:
brackets for functions should start from the same line they are introduced
eg. `function foo() {`